### PR TITLE
Fix spec for brod_utils:delete_topics

### PR DESCRIPTION
**Pull request by hrubi**

### CODE REVIEW

File: src/brod_utils.erl [hunk 1]
```diff
@@ -103,14 +103,14 @@ create_topics(Hosts, TopicConfigs, RequestConfigs, ConnCfg) ->
             end).
 %% @equiv delete_topics(Hosts, Topics, Timeout, [])
 -spec delete_topics([endpoint()], [topic()], pos_integer()) ->
-        {ok, kpro:struct()} | {error, any()}.
+        ok | {ok, kpro:struct()} | {error, any()}.
```
Comment (zmstone) [inline src/brod_utils.erl]: In which case does it return an ok tuple?

Comment (hrubi) [reply src/brod_utils.erl]: In none. It always returns either `ok` or the error tuple due to this:
```
parse(delete_topics, _, Msg) ->
  ok = throw_error_code(kpro:find(responses, Msg));
```
I will update the spec both here and in `brod:delete_topics`.

**Comment by zmstone** [comment]
@hrubi thank you for the PR.
could you help to update the changelog.md ? add a new version at the top (3.16.3)

**Comment by hrubi** [comment]
@zmstone sure. I've updated the changelog.

**Pull request merged**